pages/mesosphere/dcos/services/edge-lb/1.2/api-reference/edgelb/index.md

```diff
@@ -44,7 +44,7 @@ The Edge-LB API also requires authorization via the following permissions:
 |-------|----------|
 | `/service/edgelb/` | `dcos:adminrouter:service:edge-lb` |
 
-All routes may also be reached by users with the `dcos:superuser` permission.
+Users with the dcos:superuser permissions may reach all routes.
```

Comment:
I'd keep the formatting... s/dcos:superuser/`dcos:superuser`

Reply:
whoops, indeed 👍